assets/state-mig-manager/0400_configmap.yaml

```diff
@@ -201,9 +239,24 @@ data:
           mig-devices:
             "7g.96gb": 1
 
+      # H200-141GB
+      all-7g.141gb:
+        - devices: add
+          mig-enabled: true
+          mig-devices:
+            "7g.141gb": 1
 
-      # H100-96GB, GH200, H100 NVL, H800 NVL, H100-80GB, H800-80GB, A800-40GB, A800-80GB, A100-40GB, A100-80GB, A30-24GB, PG506-96GB
+      # H200-141GB, H100-96GB, GH200, H100 NVL, H800 NVL, H100-80GB, H800-80GB, A800-40GB, A800-80GB, A100-40GB, A100-80GB, A30-24GB, PG506-96GB
       all-balanced:
+        # H200 141GB
+        - device-filter: ["0x233510DE"]
+          devices: all
+          mig-enabled: true
+          mig-devices:
+            "1g.18gb": 2
+            "2g.35gb": 1
+            "3g.71gb": 1
```

Comment:
This may not be a valid configuration. The total memory adds up to 142GB. We may need to limit this to just one `1g.18gb` instance instead of two.

Reply:
Ah okay, thanks for pointing me to the official table. Just make sure to verify this config can be applied correctly.